docs/source/module_intro/core_module.rst

```diff
@@ -0,0 +1,16 @@
+Differentiable Computer Vision
+================================
+
+It consists of a set of routines and differentiable modules to solve generic computer vision problems. At its core, the package uses PyTorch as its main backend both for efficiency and to take advantage of the reverse-mode auto-differentiation to define and compute the gradient of complex functions.
```

Comment:
TBD: refer as a new generation computer vision library within the software 2.0 paradigm that seeks to provide differentiable implementations for classical computer vision and 2d image processing and 3d volumetric imaging algorithms.
TBD: make a itemized list with the key concepts